alchimia/__init__.py

```diff
@@ -1,10 +1,12 @@
 from __future__ import absolute_import, division
 
-from alchimia.strategy import TwistedEngineStrategy, TWISTED_STRATEGY
+from alchimia.strategy import TWISTED_STRATEGY
+from alchimia.engine import TwistedEngine
 
+wrap_engine = TwistedEngine.from_sqlalchemy_engine
+del TwistedEngine
 
 __all__ = [
     "TWISTED_STRATEGY",
+    "wrap_engine",
 ]
-
-TwistedEngineStrategy()
```

Comment:
Did you remove this to shrink the effective public API?  If so :+1:

Reply:
Yeah, it was not present in `__all__` and was not documented so I'm pretty sure it was not intentionally exported.